co2eq/countries.json

```diff
@@ -0,0 +1,416 @@
+{
+    "countries":{
+        "AF":{},
+        "AX":{},
+        "AL":{},
+        "DZ":{},
+        "AS":{},
+        "AD":{},
+        "AO":{},
+        "AI":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
```

Comment:
can you rename to `hotelFootprintKilogramsPerHotelNight` (camel case is the convention for JSON/javascript)? That will be the last change I promise 🙏  Thank you so much for your help

Reply:
Of course, I don't even know why I put a capital letter there in the first place.
You're welcome! I think it is a very interesting and helpful project. I hope to see the app out soon. :)